src/pages/chainlink-nodes/v1/using-forwarder.md

```diff
@@ -0,0 +1,353 @@
+
+   ```shell
+   echo "ETH_USE_FORWARDERS=true
+   FEATURE_LOG_POLLER=true" >> ~/.chainlink-sepolia/.env
+   ```
+
+   - `ETH_USE_FORWARDERS` enables sending transactions through forwarder contracts.
+   - `FEATURE_LOG_POLLER` enables polling forwarder contracts logs to detect any changes to the authorized senders.
+
+1. Start the Chainlink node:
+
+   ```shell
+   cd ~/.chainlink-sepolia && docker run --name chainlink  -v ~/.chainlink-sepolia:/chainlink -it --env-file=.env -p 6688:6688 smartcontract/chainlink:1.12.0 local n
+   ```
+
+1. You will be prompted to enter the key store password:
+
+   ```shell
+   2022-12-22T16:18:04.706Z [WARN]  P2P_LISTEN_PORT was not set, listening on random port 59763. A new random port will be generated on every boot, for stability it is recommended to set P2P_LISTEN_PORT to a fixed value in your environment config/p2p_v1_config.go:84       logger=1.10.0@aeb8c80.GeneralConfig p2pPort=59763
+   2022-12-22T16:18:04.708Z [DEBUG] Off-chain reporting disabled                       chainlink/application.go:373     logger=1.10.0@aeb8c80
+   2022-12-22T16:18:04.709Z [DEBUG] Off-chain reporting v2 disabled                    chainlink/application.go:422     logger=1.10.0@aeb8c80
+   Enter key store password:
+   ```
+
+1. Enter the key store password and wait for your Chainlink node to start.
+
+## Create directRequest jobs
+
+This section is similar to the [Fulfilling Requests](/chainlink-nodes/v1/fulfilling-requests) guide.
+
+1. Open the Chainlink Operator UI.
+1. On the **Jobs** tab, click **New Job**.
+1. Create the `uint256` job. Replace:
+
+   - `YOUR_OPERATOR_CONTRACT_ADDRESS` with the address of your deployed operator contract address.
+   - `EOA_ADDRESS` with the **first** Chainlink node EOA.
+
+     <CodeSample src="samples/ChainlinkNodes/forwarder/get-uint256.toml"/>
+
+1. Create the `string` job. Replace:
+
+   - `YOUR_OPERATOR_CONTRACT_ADDRESS` with the address of your deployed operator contract address.
+   - `EOA_ADDRESS` with the **second** Chainlink node EOA.
+
+     <CodeSample src="samples/ChainlinkNodes/forwarder/get-string.toml"/>
+
+1. After you create the jobs, record the two job IDs.
+
+:::note
+Note that both jobs have the attribute `forwardingAllowed = true`. This attribute will make the jobs attempt to run through the forwarder before falling back to default mode. For example, if the Chainlink node tracks no forwarder, the job falls back to the default mode.
+:::
+
+## Test the transaction-sending strategy
+
+### Create API requests
+
+1. Open [APIConsumerForwarder.sol](https://remix.ethereum.org/#url=https://docs.chain.link/samples/APIRequests/APIConsumerForwarder.sol) in the Remix IDE.
+
+1. Note that `setChainlinkToken(0x779877A7B0D9E8603169DdbD7836e478b4624789)` is configured for _Sepolia_.
+
+1. On the **Compiler** tab, click the **Compile** button for `APIConsumerForwarder.sol`.
```

Comment:
```suggestion
1. On the **Compiler** tab, click the **Compile** button for `APIConsumerForwarder.sol`.
```

Previous sections use _italics_ when describing UI elements. Either bold or italics is fine, but we should be consistent. I recommend using bold ** for UI objects.